talentsearch/.storybook/preview.js

```diff
@@ -2,6 +2,7 @@ import "../resources/css/hydrogen.css"
 import "../resources/css/app.css"
 import "../resources/js/components/IntlContainer";
 import { setIntlConfig, withIntl } from 'storybook-addon-intl';
+import frCompiled from "../resources/js/lang/frCompiled.json";
```

Comment:
If this is required for storybook now, maybe we could just auto-execute it?

```json5
  "scripts": {
    "prestorybook": "npm run intl-compile",
    "prebuild-storybook": "npm run intl-compile",
    // ...
```

Then we don't need to explicitly add it as pre-command in CI either.

But if it's slow, maybe not worth it :)

Reply:
How have I never known about npm pre- and post- scripts??? 😲

Reply:
But chromatic only really runs `npm run storybook` in the admin project, while all three need to compile their language files.